python/lsst/the/monster/utils.py

```diff
@@ -126,3 +127,107 @@ def makeRefCat(refSchema, refTable, survey, bands, reference_name):
         refCat[colname_err][:] = refTable[colname_err]
 
     return refCat
+
+
+def makeMonsterSchema(gaia_catalog_columns, bands, output_system='lsst'):
+    """
+    Make the monster refcat schema. Include all columns from Gaia, as well
+    as transformed fluxes, flux errors, and flags identifying the source
+    of each flux entry.
+
+    Parameters
+    ----------
+    gaia_catalog_columns : `List` of `TableColumns`
+        Gaia catalog columns (e.g., from "gaia_stars_all.itercols()")
+    bands : `List` of `str`
+        Names of the bands to include in the monster refcat
+    output_system : `str`
+        Name of the output system to use.
+
+    Returns
+    -------
+    monsterSchema: `lsst.afw.table.Schema`
+    """
+
+    monsterSchema = afwTable.SimpleTable.makeMinimalSchema()
+
+    exclude_columns = ["id", "coord_ra", "coord_dec", "monster_lsst_u_flux",
```

Comment:
Should this be `output_system` for the u?

Reply:
We can also programmatically build this by looping over bands, would be a bit easier to read.

Reply:
Finally, I don't understand what's going on here.  Why are we excluding these?  Does this mean they aren't getting into the output catalog?  No, they're below.  Some comments here on what is going on will be helpful.

Reply:
Sorry - that was confusing. Hopefully the comments that I added clarify things?
`    # We want to transfer all the existing Gaia columns to the output table,
    # as well as the newly-added fluxes, errors, and flags. The "transfer" loop
    # below takes care of adding the Gaia columns to the schema. 
    # Columns "id", "coord_ra", and "coord_dec" already exist in the minimal
    # schema that we have initialized, so we don't need to include them in the
    # loop to transfer entries. Also, the flux/error/flag columns don't have
    # units in the input table, so we'll treat them separately to add the
    # units. Thus we exclude them from the "transfer" loop as well.
 `